BusBoard.Web/Views/Home/BusInfo.cshtml

```diff
@@ -1,10 +1,72 @@
-﻿@model BusBoard.Web.ViewModels.BusInfo
+﻿@using Microsoft.AspNetCore.Html
+@using BusBoard.Api.Tfl
+@using BusBoard.Api.Tfl.Responses
+@using BusBoard.Web.ViewModels
+@model BusBoard.Web.ViewModels.BusInfo
 @{
-  ViewBag.Title = "Bus Information";
+    ViewBag.Title = "Bus Information";
+}
+
+@functions {
+
+  static string FormatMinutes(int minutes) => minutes == 1 ? "1 min" : $"{minutes} mins";
```

Comment:
It's more common to define helper functions like this in the view model 🙂

Reply:
That makes sense, moved in 418a34d 🙂